src/index.html

```diff
@@ -4,10 +4,78 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet">
+
+    <link rel="shortcut icon" href="./images/icons/favicon.png">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Miami</h1>
+
+    <header class="header">
+
+      <div class="header__top">
+
+        <nav class="nav">
+          <div class="nav__icon"></div>
+          <div class="nav__menu">
+            <ul class="nav__items">
+              <li class="nav__item">Home</li>
```

Comment:
```suggestion
              <li class="nav__item"><a href="...">Home</a></li>
```
There's should be links inside